gui/slick/views/snatchSelection.mako

```diff
@@ -359,7 +359,7 @@
                             <img src="${srRoot}/images/providers/missing.png" width="16" height="16" style="vertical-align:middle;" alt="missing provider" title="missing provider"/> ${hItem["provider"]}
                         % endif
                     </td>
-                    <td align="center">${renderQualityPill(int(hItem["quality"]))}</td>
+                    <td align="center">${renderQualityPill(int(hItem["quality"]))} ${('', '<span class="quality Proper">P</span>')[len(hItem["proper_tags"]) > 0]}</td>
```

Comment:
Why the len()? Just check for existence:
`('proper', '')[not hItem['proper_tags']]